v2.1/build-a-ruby-app-with-cockroachdb.md

```diff
@@ -39,11 +39,11 @@ Now that you have a database and a user, you'll run code to create a table and i
 
 First, use the following code to connect as the `maxroach` user and execute some basic SQL statements, creating a table, inserting rows, and reading and printing the rows.
```

Comment:
This sentence confuses me. When I parse the sentences: "First, use the following code..." followed by "Download the file...", I first try to find the code to connect using the maxroach user (and so on), and only then am I supposed to download the file.

I suggest we use the phrasing in the ActiveRecord section: "The following code uses...." to describe what the code in the file does.